src/main/java/org/folio/circulation/domain/policy/ExpirationDateManagement.java

```diff
@@ -0,0 +1,53 @@
+package org.folio.circulation.domain.policy;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+/**
+ * Value for hold shelf expiration date from service point
+ */
+public enum ExpirationDateManagement {
+
+  /**
+   * Long-term loans:
+   * Loan period = Months|Weeks|Days
+   * value="KEEP_THE_CURRENT_DUE_DATE", Keep the current due date</option>
+   * value="MOVE_TO_THE_END_OF_THE_PREVIOUS_OPEN_DAY", Move to the end of the previous open day
+   * value="MOVE_TO_THE_END_OF_THE_NEXT_OPEN_DAY", Move to the end of the next open day
+   */
```

Comment:
You want to divide them up into two groups. But I think you're also using a javadoc comment `/**` to do that. This could be confusing when someone mouses over the item in their IDE for example. You probably should choose either to use javadoc here, and then provide a javadoc on each item, perhaps saying which group each belongs to in the doc for each one, or just convert these to regular comments like `/*`.